docs/user/exams/instructors_guide.rst

```diff
@@ -0,0 +1,192 @@
+===================
+Instructors’ Guide
+===================
+
+.. _timeline:
+
+Timeline of an Artemis Online Exam
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+    .. _exam_creation_and_configuration:
```

Comment:
Please add a 5. step, which would be "Exam Complaint Assessment": Shortly mention that complaints of students are addressed in this step.
Please add a 6. step, which would be "Publication of Results": Shortly mention the exam scores page with statistics and the CSV export

It would also be nice, if could show a small image of this timeline with horizontal activities and arrows at the top:

Exam Creation and Configuration --> Exam Conduction --> Exam Assessment --> Student Review --> Exam Complaint Assessment --> Publication of Results